src/Views/Textile/Step.elm

```diff
@@ -185,6 +187,59 @@ dyeingMediumField { inputs, updateDyeingMedium } =
         ]
 
 
+spinningProcessField : Config msg -> Html msg
+spinningProcessField { inputs, updateMaterialSpinning } =
+    li [ class "list-group-item d-flex align-items-center gap-2" ]
+        [ div [ class "d-flex flex-column gap-1 w-100" ]
+            (inputs.materials
+                |> List.map
+                    (\{ material, spinning } ->
+                        let
+                            availableSpinningProcesses =
+                                Material.getAvailableSpinningProcesses material.origin
+                        in
+                        div [ class "d-flex justify-content-between align-items-center fs-7" ]
+                            [ label
+                                [ for <| "spinning-for-" ++ Material.idToString material.id
+                                , class "text-truncate w-25"
+                                ]
+                                [ text material.shortName ]
+                            , case availableSpinningProcesses of
+                                [ spinningProcess ] ->
+                                    input
+                                        [ type_ "text"
+                                        , class "form-control w-75"
+                                        , disabled True
+                                        , value <| Material.spinningToLabel spinningProcess
+                                        ]
+                                        []
```

Comment:
This is a bit weird looking from a UX standpoint as one may think that's a text input while no input is allowed. Please use some greyed text instead (`text-muted` class will do the job I guess)

Reply:
<img width="423" alt="Capture d’écran 2023-08-29 à 16 23 40" src="https://github.com/MTES-MCT/ecobalyse/assets/167767/a3891b64-f9ce-4741-a4e0-ed82f0e57c42">
 What do you think?